examples/src/demos/dev/Dom.js

```diff
@@ -1,8 +1,9 @@
 import React, { Suspense, createRef, useEffect, useRef } from 'react'
-import { Canvas, Dom, useFrame } from 'react-three-fiber'
+import { Html } from 'drei'
```

Comment:
Updated since `Dom` is no longer a thing in `react-three-fiber` and was delegated to `drei`